addons/binding/org.openhab.binding.icloud/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,138 @@
+			<channel typeId="Location" id="location" />
+			<channel typeId="LocationAccuracy" id="locationAccuracy" />
+			<channel typeId="LocationLastUpdate" id="locationLastUpdate" />
+			<channel typeId="DistanceFromHome" id="distanceFromHome" />
+			<channel typeId="AddressStreet" id="addressStreet" />
+			<channel typeId="AddressCity" id="addressCity" />
+			<channel typeId="AddressCountry" id="addressCountry" />
+			<channel typeId="FormattedAddress" id="formattedAddress" />
+			<channel typeId="DeviceName" id="deviceName" />
+		</channels>
+
+		<representation-property>deviceId</representation-property>
+
+		<config-description>
+			<parameter name="deviceId" type="text" required="true">
+				<label>Device id</label>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="NumberOfDevices">
+		<item-type>Number</item-type>
+		<label>Number of devices</label>
+		<description>Number of registered devices with this iCloud account.</description>
+	</channel-type>
+	<channel-type id="Owner">
+		<item-type>String</item-type>
+		<label>Owner</label>
+		<description>Registered owner of this iCloud account.</description>
+	</channel-type>
+	<channel-type id="Refresh">
+		<item-type>Switch</item-type>
+		<label>Refresh</label>
+		<description>Update all devices registered with this account immediately.</description>
+	</channel-type>
+	<channel-type id="EnableAddressLookup">
+		<item-type>Switch</item-type>
+		<label>Enable address lookup</label>
+		<description>Enable address lookup. The coordinates are send to google to lookup a
+	    human readable address.</description>
+	</channel-type>
+	<channel-type id="BatteryStatus">
+		<item-type>String</item-type>
+		<label>Battery Status</label>
+	</channel-type>
+	<channel-type id="FindMyPhone">
+		<item-type>Switch</item-type>
+		<label>Find my phone</label>
+	</channel-type>
+	<channel-type id="Location">
+		<item-type>Location</item-type>
+		<label>Location</label>
+	</channel-type>
+	<channel-type id="LocationAccuracy">
+		<item-type>Number</item-type>
+		<label>Location accuracy</label>
+		<state readOnly="true" pattern="%d m" />
+	</channel-type>
+	<channel-type id="DistanceFromHome">
+		<item-type>Number</item-type>
+		<label>Distance from home</label>
+		<state readOnly="true" pattern="%d m" />
+	</channel-type>
+	<channel-type id="LocationLastUpdate">
+		<item-type>DateTime</item-type>
+		<label>Last location update</label>
+	</channel-type>
+	<channel-type id="AddressStreet">
+		<item-type>String</item-type>
+		<label>Street</label>
+	</channel-type>
+	<channel-type id="AddressCity">
+		<item-type>String</item-type>
+		<label>City</label>
+	</channel-type>
+	<channel-type id="AddressCountry">
+		<item-type>String</item-type>
+		<label>Country</label>
+	</channel-type>
+	<channel-type id="FormattedAddress">
+		<item-type>String</item-type>
+		<label>Formatted address</label>
+	</channel-type>
+	<channel-type id="DeviceName">

```

Comment:
Should be a property, not a channel

Reply:
How/is a property also available to rules and sitemaps? If so, do you have an example I could use in the readme?

Reply:
No, but as it is static, you can also use a static string within sitemaps or rules.